exercises/pangram/canonical-data.json

```diff
@@ -1,68 +1,87 @@
 {
   "exercise": "pangram",
   "comments": [
-    "A pangram is a sentence using every letter of the alphabet at least once."
+    "A pangram is a sentence using every letter of the alphabet at least once.",
+    "",
+    "It is the track's responsibility to add further language specific tests, like:",
+    "input error handling (null/undefined parameters etc.)",
+    "referential transparency (i.e. evaluating, a function/method gives the same value for same arguments every time.)",
+    "etc.",
+    "",
+    "'error' and 'null' should be treated according to the languages' specifics and possibilities."
```

Comment:
this is not strictly necessary, since there are no longer any `error` or `null` used in this file (unless I missed one, sorry!)